pkg/aws/dynamo_locking.go

```diff
@@ -14,7 +16,39 @@ type DynamoDbLock struct {
 	DynamoDb *dynamodb.DynamoDB
 }
 
+func (dynamoDbLock *DynamoDbLock) createTableIfNotExists() {
```

Comment:
I think it's misleading to call it createTableIfNotExists() given it doesn't check that, might as well just call it createTable()